libs/community/langchain_community/vectorstores/azuresearch.py

```diff
@@ -555,12 +633,13 @@ def semantic_hybrid_search(
         Args:
             query (str): The query text for which to find similar documents.
             k (int): The number of documents to return. Default is 4.
+            filters: Filtering expression.
```

Comment:
@baskaryan here and in 661: there is no argument `filters` that you added to the documentation. Is it intended?

Reply:
think its useful to document since you can pass in via **kwargs